modules/op-release-notes-1-18.adoc

```diff
@@ -385,3 +385,21 @@ spec:
 * Before this update, when using {pac}, if you used the `generateName` field in a `PipelineRun` resource to match the resource to the `incoming` webhook, the match would not work correctly. With this update, the issue is fixed.
 
 * Before this update, when using {pac}, the `on-cel-expression` annotation was not working on push events in Bitbucket Data Center. With this update, the issue is fixed.
+
+[id="release-notes-1-18-1_{context}"]
+== Release notes for {pipelines-title} 1.18.1
+
+With this update, {pipelines-title} General Availability (GA) 1.18.1 is available on {OCP} 4.15 and later versions.
+
+[id="fixed-issues-1-18-1_{context}"]
+== Fixed issues
+
+* Before this update, `Results` configuration options were not being applied to `TektonConfig` Custom Resource (CR). With this update, `Results` configuration options from the `TektonConfig` CR are now correctly applied to the TektonResults object.
+
+* Before this update, if `PipelineRun` without the `cancel-in-progress` annotation was executed due to a Pull Request (PR) and that PR was closed, the `PipelineRun` would be automatically cancelled. With this update, `Pipelines as Code` can cancel `PipelineRun` if the `cancel-in-progress` annotation is explicitly specified.
```

Comment:
```suggestion
* Before this update, if {pac} started a pipeline run without the `cancel-in-progress` annotation because of a Pull Request (PR) and that PR was closed, the pipeline run was automatically cancelled. With this update, {pac} does not cancel the pipeline run unless the `cancel-in-progress` annotation is explicitly specified.
```